GANDLF/models/unetr.py

```diff
@@ -1,32 +1,26 @@
 from .modelBase import ModelBase
 import sys
-from GANDLF.utils.generic import checkPatchDimensions
 from GANDLF.models.seg_modules.out_conv import out_conv
 import torch
 import torch.nn as nn
 import numpy as np
 import math
 
+
 class _DeconvConvBlock(nn.Sequential):
-    def __init__(
-        self,
-        in_feats,
-        out_feats,
-        Norm,
-        Conv
-    ):
+    def __init__(self, in_feats, out_feats, Norm, Conv):
         super().__init__()
-        
+
         self.add_module(
             "deconv",
             nn.ConvTranspose3d(
```

Comment:
We could pass an additional `ConvTranspose` to this class and use that instead of `ConvTranspose3d`. This value is defined [in `ModelBase`](https://github.com/CBICA/GaNDLF/blob/deec1896417e8f7ed7a99e6aee29a4a3f4b5cc2f/GANDLF/models/modelBase.py#L52). 😄